designer/pydesigner.py

```diff
@@ -458,6 +464,86 @@
         filetable['HEAD'] = filetable['undistorted']
 
 #---------------------------------------------------------------------- 
+# Create Brain Mask
+#----------------------------------------------------------------------
+if args.mask:
+    fsl_suffix = '.gz'
+    brainmask_fsl_name = 'brain'
+    brainmask_fsl_full = op.join(outpath, brainmask_fsl_name)
+    brainmask_fsl_out = op.join(outpath, brainmask_fsl_name + '_mask' +
+    '.nii' + fsl_suffix)
+    brainmask_out = op.join(outpath, brainmask_fsl_name + '_mask' + '.nii')
+    B0_name = 'B0.nii'
+    B0_mean = 'B0_mean.nii'
+    B0_full = op.join(outpath, B0_name)
+    B0_mean_full = op.join(outpath, B0_mean)
+    # check to see if this already exists
+    if (op.exists(brainmask_out) and not args.resume):
+            raise Exception('Running mask would cause an overwrite. '
+                            'In order to run please delete the files, use '
+                            '--force, use --resume, or change output '
+                            'destination.')
+    if not (args.resume and op.exists(brainmask_out)):
+        if args.maskthr is None:
+            maskthr = 0.25
+        else:
+            maskthr = args.maskthr
+        # Extract B0s
+        print(filetable['HEAD'].getFull())
```

Comment:
Please add a verbosity check first.

Reply:
Was one of my debugging print. Removed it in new pull request